crates/cache/Cargo.toml

```diff
@@ -7,10 +7,8 @@ edition = "2021"
 moon_config = { path = "../config"}
 moon_error = { path = "../error"}
 moon_logger = { path = "../logger"}
-moon_project = { path = "../project"}
```

Comment:
Pulling the cache engine into the project crate caused a cycle, so I moved the hasher to its own crate.